packages/affiliates/liquidity-mining/yusdbtc-weekly-payouts/Week_29_Mining_Rewards.json

```diff
@@ -0,0 +1,97 @@
+{
+  "week": 29,
+  "fromBlock": 12091493,
+  "toBlock": 12136988,
+  "poolAddress": ["0x6be6258fe363288b397882c071531b3623fd5fd9", "0x29fcd0d34477f4cac604a525cb5cf1065fe14502"],
+  "blocksPerSnapshot": 256,
+  "umaPerWeek": 2325,
+  "shareHolderPayout": {
```

Comment:
this `shareHolderPayout` contains the sum of the roll between the two contracts.👇